src/FakeItEasy/Creation/IProxyOptions.cs

```diff
@@ -12,6 +12,6 @@ internal interface IProxyOptions
 
         IEnumerable<Action<object>> ProxyConfigurationActions { get; }
 
-        IEnumerable<CustomAttributeBuilder> AdditionalAttributes { get; }
+        IEnumerable<Expression<Func<Attribute>>> AdditionalAttributes { get; }

```

Comment:
I see we've retained the "additional" here. It's only internal, so not crucial, but do we want to more closely match the language of the public API?